pom.xml

```diff
@@ -19,13 +19,27 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>com.ibm.websphere.azure</groupId>
-    <artifactId>arm-rhel-was-nd-cluster-root</artifactId>
-    <packaging>pom</packaging>
-    <version>1.2.5</version>
+    <artifactId>azure.websphere-traditional.cluster</artifactId>
+    <version>1.3.1</version>
+    
+    <parent>
+        <groupId>com.microsoft.azure.iaas</groupId>
+        <artifactId>azure-javaee-iaas-parent</artifactId>
+        <version>1.0.7</version>
+        <relativePath></relativePath>
+    </parent>
+    
+    <packaging>jar</packaging>
     <name>${project.artifactId}</name>
 
-    <modules>
-      <module>arm-rhel-was-nd-cluster</module>
-      <module>add-nodes</module>
-    </modules>
+    <properties>
+        <test.parameters>-TestParameter '@{&quot;PasswordMinLength&quot;=6}'</test.parameters>
+        <azure.apiVersion>2020-06-01</azure.apiVersion>
+        <azure.apiVersion2>2019-06-01</azure.apiVersion2>
+        <cluster.start>c2efe274-0dc2-50f3-b51f-338a3fe01afb</cluster.start>
+        <cluster.end>9c5dbfce-a6b9-5659-96bc-ca0cf429122a</cluster.end>
+        <image.publisher>microsoft_javaeeonazure_test</image.publisher>
+        <image.offer>2021-zheng-image-offer-preview</image.offer>
+        <image.sku>2021-zheng-rhel-with-websphere</image.sku>
```

Comment:
Note: these values will be updated once the official VM base image is published.

Reply:
Created issue https://github.com/WASdev/azure.websphere-traditional.cluster/issues/11 to track this.